editor/debugger/editor_profiler.cpp

```diff
@@ -633,6 +639,13 @@ EditorProfiler::EditorProfiler() {
 	clear_button->set_disabled(true);
 	hb->add_child(clear_button);
 
+	autostart_checkbox = memnew(CheckBox);
+	autostart_checkbox->set_toggle_mode(true);
+	autostart_checkbox->set_text(TTR("Autostart"));
+	autostart_checkbox->set_pressed(EditorSettings::get_singleton()->get_project_metadata("debug_options", "autostart_profiler", false));
+	autostart_checkbox->connect("pressed", callable_mp(this, &EditorProfiler::_autostart_pressed));
```

Comment:
I know _nothing_ about godot codebase, but looks like `"pressed"` should be replaced with `SceneStringName(pressed)`